experimental/PropertyDDS/packages/property-common/package.json

```diff
@@ -82,7 +82,7 @@
     "cross-env": "^7.0.2",
     "eslint": "~8.6.0",
     "mocha": "^10.0.0",
-    "nock": "^10.0.1",
+    "nock": "^13.3.3",
```

Comment:
it seems odd to update a bunch of non server things on a server release branch.

Reply:
I agree and was thinking how can we do this better. This is from cherry-picking a commit that fixed lts to be able to run most of our pipelines. While a good chunk of it was about client and azure, I was hoping not to have to disentangle that PR.